src/go/rpk/pkg/cli/registry/schema/check_compatibility.go

```diff
@@ -66,7 +66,8 @@ func newCheckCompatibilityCommand(fs afero.Fs, p *config.Params) *cobra.Command
 				Type:       t,
 				References: references,
 			}
-			compatible, err := cl.CheckCompatibility(cmd.Context(), subject, version, schema)
+			ctx := sr.WithParams(cmd.Context(), sr.Verbose)
+			compatible, err := cl.CheckCompatibility(ctx, subject, version, schema)
```

Comment:
I had a feeling that it was previously decided to always report verbose, but I might be wrong.

I think being able to disable verbose is useful.

Reply:
Yes, here: https://github.com/redpanda-data/redpanda/pull/18332#discussion_r1603753908

> I think being able to disable verbose is useful.

What use case do you have in mind? My main reason against is that I'm not sure what to call the flag because there's already a global flag `--verbose` with rpk and the two might be confusing together.

cc @twmb @r-vasquez what do you think?

Reply:
>not sure what to call the flag

could just go with `--quiet` or something and map that to `"verbose": false`. but yeah, i don't have a strong opinion wrt how useful that is in practice.

Reply:
> What use case do you have in mind?

Mostly in testing the `verbose` flag and whether it returns `messages`? But that's easy to achieve with `requests` module, so not a reason to change the implementation of rpk.

Reply:
Yeah I could test with just requests so I'll leave this for the rpk team to decide.